pkg/policy/trafficdirection/trafficdirection.go

```diff
@@ -19,13 +19,13 @@ type TrafficDirection uint8
 
 const (
 	// Invalid represents an invalid traffic direction.
-	Invalid TrafficDirection = 2
+	Invalid TrafficDirection = 0
```

Comment:
This is a subpackage under policy, are they the same values? Maybe the metrics stuff just shouldn't be using the policy directionality defines.

Reply:
yep also @aanm pointed out it need to do per cpu aggregation correctly as well. Marking this entire thing wip for now.